lib/src/builder/execute/optimism.rs

```diff
@@ -101,6 +102,7 @@ impl TxExecStrategy<OptimismTxEssence> for OpTxExecStrategy {
                 blk_env.gas_limit = block_builder.input.gas_limit;
             })
             .with_db(block_builder.db.take().unwrap())
+            .append_handler_register(optimism::optimism_handle_register)
```

Comment:
So, in the newest revm, we must register handler ourself?

Reply:
It's a new for other chains too.

Reply:
> So, in the newest revm, we must register handler ourself?

I have no idea, and have asked the `revm` developers the same question. My current guess is that (at least for OP) this should happen automatically when the feature is enabled.
However, if support for other chains is developed outside the `revm` repo, this is the new way to register them.